core/domain/exp_services.py

```diff
@@ -2215,7 +2414,7 @@ def are_changes_mergeable(exp_id, frontend_version, change_list):
                 continue
             else:
                 changes_are_mergeable = False
-                return False
+                break
```

Comment:
Why this change?

Reply:
Yeah, actually I got a lint error where I cannot use more than 6 returns in a function. Therefore change this last return to break as the break will do the same job here.